openapi/components/schemas/Cashier/CashierRequest.yaml

```
@@ -58,6 +58,35 @@ properties:
     description: Date and time at which the cashier request expires. The default expiration time is one hour from the time the request is created.
     type: string
     format: date-time
+  propertiesSchema:
+    readOnly: true
+    description: |-
+      Defines properties the user can complete when they use the hosted cashier app..
```

nitpick: avoid using `app` unless referring to the Rebilly app
```suggestion
      Defines properties the user can complete when they use the hosted cashier form.
```